bin/post-merge.js

```diff
@@ -0,0 +1,16 @@
+// MIT © Sindre Sorhus - sindresorhus.com
+// via https://gist.github.com/sindresorhus/7996717
+
+const execFile = require("child_process").execFile;
+
+execFile("git diff-tree",
+  ["-r", "--name-only", "--no-commit-id", "ORIG_HEAD HEAD"],
```

Comment:
My upgrade is taking some time on my test machine. I have an icky feeling `"ORIG_HEAD HEAD"` here should be `"ORIG_HEAD", "HEAD"`. I'll test to verify once my upgrade finishes.